docs/nginx-configuration.md

```diff
@@ -1,8 +1,33 @@
 # Nginx Configuration
 
-Altis Cloud uses Nginx for the web server which is responsible for serving static files, and proxying requests to PHP for dynamic content. For advanced customization we support additional Nginx configuration via the project codebase.
+Altis Cloud uses Nginx within your [web containers](./architecture.md). For advanced customization we support additional Nginx configuration via the project codebase.
 
-Nginx will look for an additional file in your project repository at `.config/nginx-additions.conf` and will load it into the `server {}` context of the main configuration.
+**Note:** Custom configuration will only apply to traffic served by the web container, which does not include media or tachyon (see the [architecture diagram for more information](./architecture.md)). Specifically, any URLs beginning with `/uploads/` or `/tachyon/` are routed directly to [S3](./s3-storage.md) and [Tachyon](docs://media/dynamic-images.md) directly.
+
+
+## Providing Configuration
+
+Nginx will look for an additional file in your project repository at `.config/nginx-additions.conf` and will load it into the `server {}` context of the main configuration. You can also use filename suffixes to split your configuration in complex cases.
+
+Internally, the Altis Nginx configuration looks like:
+
+```
+server {
```

Comment:
We also recently added for https://github.com/humanmade/docker-wordpress-nginx/pull/62 `nginx-http-additions` so you can do things like use `map` etc.